portal/server/utils/csv-utils.test.js

```diff
@@ -21,6 +22,20 @@ describe('csv-utils', () => {
     });
   });
 
+  describe('excelColumnToColumnIndex', () => {
+    it('returns the correct index for a column below 26', async () => {
+      const columnIndex = excelColumnToColumnIndex('C');
+
+      expect(columnIndex).toBe(2);
+    });
+
+    it('returns the correct index for a column above 26', async () => {
+      const columnIndex = excelColumnToColumnIndex('AE');
+
+      expect(columnIndex).toBe(30);
+    });
+  });
+
```

Comment:
Could we add any tests to check the new logic does what we expect (adding empty strings to empty cells)

Reply:
Mm yes I definitely should have just added these, will add something :)